webapp/src/containers/createIncident/index.tsx

```diff
@@ -218,6 +231,12 @@ const CreateIncident = () => {
         }
     }, [open, refetchIncidentFields]);
 
+    useEffect(() => {
+        if (channel) {
```

Comment:
Isn't their any other `useEffect` in which you can add this logic?

Reply:
no, this is the second useEffect used in this component